src/common/colorspaces.c

```diff
@@ -439,12 +439,12 @@ static cmsHPROFILE dt_colorspaces_create_brg_profile()
   return profile;
 }
 
-static cmsHPROFILE dt_colorspaces_create_gamma_rec709_rgb_profile(void)
+static cmsHPROFILE dt_colorspaces_create_rec709_rgb_profile(void)
 {
   cmsFloat64Number srgb_parameters[5] = { 1/0.45, 1.0 / 1.099,  0.099 / 1.099, 1.0 / 4.5, 0.081 };
   cmsToneCurve *transferFunction = cmsBuildParametricToneCurve(NULL, 4, srgb_parameters);
 
-  cmsHPROFILE profile = _create_lcms_profile("Gamma Rec709 RGB", "Gamma Rec709 RGB",
+  cmsHPROFILE profile = _create_lcms_profile("Rec709 RGB", "Rec709 RGB",
```

Comment:
Please leave the previous name untouched.

Reply:
Care to elaborate please? Is there some dependency on this originally brain-dead string? Or you changed your mind since[ ](https://discuss.pixls.us/t/please-ditch-the-g-word/18853)

Reply:
Rec 709 (https://www.itu.int/dms_pubrec/itu-r/rec/bt/R-REC-BT.709-6-201506-I!!PDF-E.pdf) defines only the RGB primaries chromaticities and is a scene-referred space (no EOTF). Used as-is, it defines the linear Rec709 color space. It defines an OETF which matters only if you are going to break RGB into Y'CbCr signal for TV broadcasting.

Rec 709 primaries are used with the sRGB EOTF to define the sRGB color space.

Rec 709 primaries are used with the ITU-R BT.1886 EOTF (power function of exponent 2.4, see https://www.itu.int/dms_pubrec/itu-r/rec/bt/R-REC-BT.1886-0-201103-I!!PDF-E.pdf) to define what we call the "gamma Rec 709".

The proper name for Gamma Rec 709 should probably be ITU-R BT.1886, and I'm pretty sure nobody will understand what it is, so "gamma" is the lesser evil here. In any case, you can't just remove the "gamma" because then you imply "linear Rec 709".

Reply:
Rec709 clearly specifies a transfer curve, slightly different to the sRGB one. No further qualification is necessary.

Reply:
Item 1.2 covers the Opto-electronic conversion (OETF). There is no EOTF specified because Rec709 is not a display-referred space (as sRGB is) and the OETF specified in BT.1886 is actually not the inverse of the BT.709 OETF but results in darkening to account for dim surround.

In all this mess, it's simply misleading and confusing to have a list composed of "linear Rec 709", "linear Rec 2020", "PQ/HLG Rec 2020", and "Rec 709" alone. The rest of the list mentions explicitely the transfer function used.

Reply:
While I might be getting this wrong, I think the Rec709 OETF and it's inverse is all we care about when we transform data between color spaces (and import/export) to keep the relationship with scene linear light. This is how the internal ICC profile is constructed as well. No gamma involved for the display correction AFAICT.

Reply:
See also https://github.com/colour-science/colour/blob/develop/colour/models/rgb/datasets/itur_bt_709.py

Reply:
I wonder if having actually two profiles would be required here? e.g. "Rec709 (reference)" for import/export using the OETF and its inverse, and "Rec709 (display)" that is using the BT.1886 EOTF assumption for soft-proofing?

Reply:
I also found [this nugget](https://www.itu.int/dms_pubrec/itu-r/rec/bt/R-REC-BT.2087-0-201510-I!!PDF-E.pdf) also offering two conversion options.

Reply:
I'll therefore drop this part of the change so this PR concerns the lut3d changes only. How Rec709 is handled becomes a separate issue then.

Reply:
> I wonder if having actually two profiles would be required here? e.g. "Rec709 (encoding)" for import/export using the OETF and its inverse, and ["Rec709 (display)" that is using the BT.1886 EOTF](https://www.color.org/rec709.xalter) assumption for soft-proofing?

This would be depend on the use case and on its context, which is why I'm always asking for a definite use case before opening an IDE.

Reply:
In the context of "we might have an actual bug here": we only have the one "gamma rec709" ICC profile defined that we use for both input decoding to scene linear (from e.g. AVIFs/HEIFs etc.) and output encoding. From the name you think is should be using BT1886 2.4 gamma as EOTF, while in fact [it is implemented w/ the inverse of the OETF](https://github.com/darktable-org/darktable/blob/9bf29b49f657fd91f72c3dbb2e88d4c8b3f444b1/src/common/colorspaces.c#L442-L444).

Reply:
Also, I just realized the same is true for PQ_* and HLG_ * color spaces as well: there is no accounting for the scene to display OOTF anywhere AFAICT. We only go to/from scene-referred and in/out encoding through OETF and its inverse, so why should (non-linear) Rec709 be any different? Again, in this context, the "gamma" label is highly misleading and should be removed or replaced with something else.